mami/script.js

```diff
@@ -4,7 +4,7 @@ console.log(x);
 let y = 500 * 4 * 12;
```

Comment:
The second answer is wrong. There is 52 weeks per year and not 48! 
The solution is : 
let y = 500 * 52  which the result is 26k. But your result is 24k a year!